index.html

```diff
@@ -17,7 +17,7 @@
     <section class="">
```

Comment:
- [ ] On line 8 of this file, change the page's title to a better and meaningful name. You may choose to use Mutinhiri's portfolio